.github/workflows/CI.yml

```diff
@@ -33,8 +33,8 @@ jobs:
 
       - name: Install Firefox Headless
```

Comment:
probably this is not needed, will see if we can remove it.

Reply:
removed with commit https://github.com/webarkit/jsartoolkitNFT/pull/239/commits/59b4f64e1b2cd62efd5bf921653974d806b0e422 CI is ok, not needed to install the package.